script/lib/check

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+
+LIB_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+. "$LIB_DIR/check-aws-credentials"
```

Comment:
`.` = `Source`

This ensures any actions taken inside these scripts are persisted, because they're all in the same process.

It also means that when we exit from inside these scripts, it exits the whole script.

Reply:
Would `Source` be clearer for future readers? If you don't know what `.` does, it's difficult to ask a search engine!

Reply:
Fair point! Will change that now